recipes/django-haystack/run_test.py

```diff
@@ -0,0 +1,12 @@
+# try to import haystack
```

Comment:
I don't know the first thing about `django` but maybe you can do something like https://github.com/conda-forge/django-test-plus-feedstock/blob/master/recipe/run_test.py?

I have also seen some 1-liners that add a Django context to make this work. I'll try to find an example for you.

Reply:
Good point @ocefpaf! it is worth a shot